### DISCUSSION TITLE
WebHelp - missing NavBar

### DISCUSSION THREAD

**Discussion by aevola** [EWSoftware/SHFB#963] [Q&A] [answered]
Hello,

I want create a C# documentation for a project. After some searchs on internet I found SHFB to auto-generate doc. from C# comments.
It's exactly what I need, thanks for this wonderfull work 👍 

But, in spite of all my efforts to understand how it work, I'm stuck...

Indeed, I follow some tutorials, videos, watch your guide on your website ([SHFB/Welcome](https://ewsoftware.github.io/SHFB/html/bd1ddb51-1c4f-434f-bb1a-ce2135d3a909.htm)), and tried to modify some "user's preferences" into the SHFB project properties (into VS and/or into the app GUI).
<b>Every times I make the Web documentation, the navigation bar is missing.</b> (cf the screen shot below)
Is-it normal ? 
Is it possible to generate it ? If yes, how ?
![Capture d’écran 2023-01-26 161408](https://user-images.githubusercontent.com/15126263/214875665-9f16e141-1818-44e0-aaf7-c0b544a14457.png)
At left : your website
At right : the documentation generate with SHFB from the sources

<b>Below the procedure that I followed :</b>
1 - Download zip installer [here](https://github.com/EWSoftware/SHFB/releases)
2 - Unzip the installer, and follow the installation
3 - Run the SHFB project
4 - Go to the output folder, and open the index.html

PS : Hoping that my english is correct, I remain at your disposal if needed

**Comment by EWSoftware** [comment, marked as answer]
Use the option in SHFB to view the web output which runs it through an instance of IIS.  If using Visual Studio, you'll find the option on the SHFB toolbar or on the View menu (View Help File).  Viewing the output from the local file system causes the browser to block all script so it won't show the TOC.

**Reply by aevola** [reply]
Thank you, that's answer my question ! =)
What's the right procedure to follow in order to make a local web documentation if the browser block all script ?

**Reply by EWSoftware** [reply]
If you're asking how to host it on the local machine without going through a web server or local instance of IIS, I don't think you can.  If neither of those is an option, you can create a Help 1 (CHM) file which is self-contained.  It's an older format but still works though it depends on the old IE control to view the content so it's days may be numbered.  You'll need to use the older VS2013 presentation style as it's the only one capable of producing CHM output.

**Reply by aevola** [reply]
OK tank very much, I've all I need 😀
